dev/archery/archery/integration/runner.py

```diff
@@ -669,6 +669,12 @@ def append_tester(implementation, tester):
                          "RenewFlightEndpoint are working as expected."),
             skip_testers={"JS", "C#", "Rust"},
         ),
+        Scenario(
+            "do_exchange:echo",
+            description=("Test the do_exchange method by "
+                         "echoing data back to the client."),
+            skip_testers={"Go", "Java", "JS", "Rust"},
```

Comment:
I have a Java implementation working too but think that should be a separate PR.